docs/deployment/2-cluster-wide-access-setup.yaml

```diff
@@ -59,6 +59,17 @@ rules:
   - update
   - delete
 
+- apiGroups:
+  - autoscaling
+  resources:
+  - horizontalpodautoscalers
+  verbs:
+  - list
```

Comment:
You could just set `*` instead of listing all verbs.

Reply:
But we want to list the needed verbs only to prevent unintended permissions. To catch bugs in code, mostly. It also serves as documentation as to what exactly is needed/used.